remix/app/components/FormInput/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import FormInput from './';
+
+jest.mock('remix-validated-form', () => ({
+  useField: jest.fn().mockReturnValue({
+    error: 'Test error',
+    getInputProps: jest.fn().mockReturnValue({}),
+  }),
+}));
+
+describe('FormInput', () => {
+  describe('givens FormInput attribute', () => {
+    it('renders the label, input, and error message', () => {
+      const label = 'Test Label';
+      const name = 'testName';
+      const placeholder = 'Test Placeholder';
+      const error = 'Test error';
+
+      render(<FormInput label={label} name={name} placeholder={placeholder} />);
+
+      const labelElement = screen.getByText(label);
+      const inputElement = screen.getByPlaceholderText(placeholder);
+      const errorElement = screen.getByText(error);
+
+      expect(labelElement).toBeInTheDocument();
+      expect(inputElement).toBeInTheDocument();
+      expect(errorElement).toBeInTheDocument();
+    });
+  });
+
+  it('enter value', () => {
```

Comment:
How about:

```suggestion
  it('sets the text input value', () => {
```

Reply:
Yes, that's better

Reply:
Improved here: [0af22fb](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/0af22fb487b5618b545e2edf90f9b9e0fe94da67)